pytorch_toolkit/ote/ote/modules/compression/nncf_config_generator.py

```diff
@@ -64,7 +66,11 @@ def __call__(self, template_path):
         if not compression_parts_to_choose:
             return {}
 
-        nncf_config_part = self._merge_nncf_compression_parts(compression_config_path, compression_parts_to_choose)
+        try:
+            nncf_config_part = self._merge_nncf_compression_parts(compression_config_path, compression_parts_to_choose)
```

Comment:
why do you make this change?

Reply:
To avoid crash of the script.
Now if the template has inconsistent format, the script prints the error, compression is not working, but crash is not happened.

I can remove this if you thing that it is not required.

Reply:
please remove, in that case we will react faster if something goes wrong

Reply:
Fixed